router/core/operation_processor.go

```diff
@@ -189,6 +190,13 @@ func (o *OperationKit) Parse(ctx context.Context, clientInfo *ClientInfo, log *z
 				parseErr = invalidExtensionsTypeError(valueType)
 				return
 			}
+			if !gjson.ValidBytes(value) {
+				parseErr = &inputError{
+					message:    "error parsing request body",
```

Comment:
Two pointa

1. Can we do this check on the whole request body instead of only for extensions?
2. If not, I'd try to be more concrete with the error message otherwise it is hard to debug.

Reply:
Adding a ticket to address it later.